how-to/how-to-use-k8s-with-cri-containerd-and-kata.md

```diff
@@ -46,20 +49,24 @@ $ command -v kubeadm
 
 ### Configure containerd to use Kata Containers
 
-The CRI containerd plugin support configuration for two runtime types.
+The CRI containerd plugin supports configuration for two runtime types.
 
-- Default runtime: A runtime that is used by default to run workloads.
-- Untrusted workload runtime: A runtime that will be used run untrusted workloads.
+- **Default runtime:** A runtime that is used by default to run workloads.
+- **Untrusted workload runtime:** A runtime that will be used run untrusted workloads.
```

Comment:
Suggested rewrite to stay active (and adding in a missing "to": **Untrusted workload runtime:** A runtime that is used to run untrusted workloads.